<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>issuebook</class>
 <widget class="QMainWindow" name="issuebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(226, 226, 226);
	color: rgb(0, 0, 0);
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
         </property>
         <property name="text">
          <string>Search Book</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="18" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(226, 226, 226);
	color: rgb(0, 0, 0);
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
         </property>
         <property name="text">
          <string>Issue</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Note</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Book ISBN</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item row="16" column="1">
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="3">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item row="18" column="3">
        <widget class="QPushButton" name="pushButton_5">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(226, 226, 226);
	color: rgb(0, 0, 0);
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Issue Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Member ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="dateEdit_2"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Return Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(226, 226, 226);
	color: rgb(0, 0, 0);
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
         </property>
         <property name="text">
          <string>Search Member</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
